File: dataset/avatar/Python/Code/codeforces_612_B.py
n = int ( input ( ) )
a = list ( map ( int , input ( ).split ( ) ) )
b = [ 0 ] * n
for i in range ( n ) :
    b [ a [ i ] - 1 ] = i
res = 0
for i in range ( 1 , n ) :
    res += abs ( b [ i ] - b [ i - 1 ] )
print ( res )

File: output/deepseek_r1/avatar/algo_based_translation/python/java/successful_translation/codeforces_612_B.java

import java.util.Scanner;

public class codeforces_612_B {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int n = scanner.nextInt();
        scanner.nextLine();
        String[] inputValues = scanner.nextLine().split(" ");
        int[] a = new int[n];
        for (int i = 0; i < n; i++) {
            a[i] = Integer.parseInt(inputValues[i]);
        }
        int[] b = new int[n];
        for (int i = 0; i < n; i++) {
            b[a[i] - 1] = i;
        }
        int res = 0;
        for (int i = 1; i < n; i++) {
            res += Math.abs(b[i] - b[i - 1]);
        }
        System.out.println(res);
        scanner.close();
    }
}

